src/runners/helpers/dbconfig.py

```diff
@@ -27,18 +27,18 @@ def format_p8_from_key(key, encrypted=False):
 
 SA_KMS_REGION = environ.get('SA_KMS_REGION', "us-west-2")
 
-# database & account properties
-REGION = environ.get('REGION', "us-west-2")
+# Database & account properties
+HOST = environ.get('SNOWFLAKE_HOST', '')
+PORT = environ.get('SA_PORT')
+PROTOCOL = environ.get('SA_PROTOCOL')
+
+REGION = environ.get('SA_REGION', "us-west-2")
 REGION_SUBDOMAIN_POSTFIX = '' if REGION == 'us-west-2' else f'.{REGION}'
-ACCOUNT = environ.get('SNOWFLAKE_ACCOUNT', '') + REGION_SUBDOMAIN_POSTFIX
+ACCOUNT = environ.get('SNOWFLAKE_ACCOUNT', '') if HOST else environ.get('SNOWFLAKE_ACCOUNT', '') + REGION_SUBDOMAIN_POSTFIX
 
 USER = environ.get('SA_USER', "snowalert") + tail
 PRIVATE_KEY_PASSWORD = environ.get('PRIVATE_KEY_PASSWORD', '').encode('utf-8')
-pk = environ.get('PRIVATE_KEY')
-PRIVATE_KEY = (
-    b64decode(pk) if pk.startswith('LS0t')  # "LS0t" is base64 of '---'
-    else format_p8_from_key(pk, encrypted=PRIVATE_KEY_PASSWORD)
-) if pk else None
```

Comment:
Looks like this is a regression to how the code was awhile ago -- can you remove this change from this PR?